docs/monitoring/MonitoringMigrationDashboardSharded.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: default
+title: Sharded Migration Monitoring Dashboard
+parent: Monitoring
+nav_order: 3
+---
+# Sharded Migration Monitoring Dashboard
+
+For a Sharded Migration a Monitoring Dashboard will be created for each shard and an Aggregated Dashboard will be created for the migration.
+
+<details open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+1. TOC
+{:toc}
+</details>
+
+## Where is my Dashboard?
+
+On the UI, the Migration Dashboards can be found under the **Monitoring Dashboards** section on the Prepare Migration page after all the resources have been generated. Along with this a list with shards and their corresponding dahsboards can be found on the Aggregated Monitoring Dashboard itself as described [here](#shards-to-dashboard).
+
+<img src="https://services.google.com/fh/files/misc/mon-dash-prep-mig-agg.png"  style="width:500px;"/>
+
+On the CLI, the unique name for each dahsboard along with the shard id will be printed on the console. These dashboards can be accessed through cloud monitoring custom dashboards page. Aggregated Monitoring Dashbaord name will also be provided.
```

Comment:
typo: dahsboard -> dashboard

Reply:
Suggest we also add a link here to say that if you don't see the link, please check your permissions and redirect there.